Libraries/vendor/core/Map.js

```diff
@@ -607,7 +620,15 @@ module.exports = (function(global, undefined) {
         }
```

Comment:
(_The error above this line, to which github is not allowing me to comment;_)

This could also fallback to the solution below.

Reply:
Good idea!